### ISSUE TITLE
Specify path for saving snapshots

### ISSUE THREAD

**Issue by Marcel-B** [theramis/Snapper#48]
Since tests sometimes have a pretty long method name and are also deeply nested in folders, PathTooLongExceptions can easily occur. 

So it would be good if you can specify a path to store Sanpshots.

**Comment by theramis** [comment]
Hey @Marcel-B, 
I'm actually looking into this at the moment. I'm thinking of introducing the concept of passing letting a user pass in a `SnapshotId` which would define the location of the snapshot.

So it would end up looking something like this. 

```
obj. ShouldMatchSnapshot(new SnapshotId(filePath, snapshotName, childSnapshotName);
```


What would you think about that?

**Comment by Marcel-B** [comment]
That looks good.  If the file path is automatically taken into the update process, this is excellent!

**Comment by theramis** [comment]
I've released an experimental feature in [2.2.4](https://www.nuget.org/packages/Snapper/2.2.4). See https://theramis.github.io/Snapper/#/pages/snapper/advanced_snapshot_control for details on how to configure where the snapshot is stored.
